### PR TITLE
Add Danger

### PR DESCRIPTION
**Pull request by orta**
Sorry ~250 watchers, I got hit by https://github.com/isaacs/github/issues/361

**Pre-requisite**

This requires the CI ENV var `DANGER_GITHUB_API_TOKEN` to be set up, so someone has to [follow this guide](http://danger.systems/guides/getting_started.html) from "Creating a GitHub Account for Danger to use". 

I'd recommend making a new GitHub account for GrapeCI - see our [DangerCI one](https://github.com/DangerCI?tab=activity), don't give it access to any of the repos. 

Then re-build this PR. I've not made any README changes, so it should show a message about me adding one.

### CODE REVIEW

**Comment by dblock** [comment]
Well, this is not going to work. 

_Travis CI makes encrypted variables and data available only to pull requests coming from the same repository. These are considered trustworthy, as only members with write access to the repository can send them. Pull requests sent from forked repositories do not have access to encrypted variables or data._

**Comment by orta** [comment]
The guide covers this - http://danger.systems/guides/getting_started.html#setting-up-an-access-token

I can amend it if it wasn't obvious?

> To get started, open https://github.com in a private browser session.

> OSS PROJECTS

> Do not add the bot to repo, or to your organization.

And later

>ADDING THE ENV VAR

> You need to add the DANGER_GITHUB_API_TOKEN environment variable, to do this, go to your repo's settings, which should look like: https://travis-ci.org/[user]/[repo]/settings.

> If you have an open source project, you should ensure "Display value in build log" enabled, so that PRs from forks work.

So long as you haven't given the account any access, the token is effectively valueless for anything but commenting.

**Comment by dblock** [comment]
I missed the "Display value in build log" part. Restarted the build.

**Comment by dblock** [comment]
The format for our changelog has a dash before `[@orta]` :)

**Comment by orta** [comment]
Cool beans, great, looks good, I've pushed a fix and a CHANGELOG entry. The Danger message should get deleted.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 * [#1366](https://github.com/ruby-grape/grape/pull/1366): Store `message_key` on `Grape::Exceptions::Validation` - [@mkou](https://github.com/mkou).
 * [#1398](https://github.com/ruby-grape/grape/pull/1398): Added `rescue_from :grape_exceptions` - allow Grape to use the built-in `Grape::Exception` handing and use `rescue :all` behavior for everything else - [@mmclead](https://github.com/mmclead).
 * [#1443](https://github.com/ruby-grape/grape/pull/1443): Extended `given` to receive a `Proc` - [@glaucocustodio](https://github.com/glaucocustodio).
+* [#1455](https://github.com/ruby-grape/grape/pull/1455): Adds some automated PR rules around CHANGELOGs and ensuring the `fit` and `fdescribe` don't end up in the test suite. - [@orta](https://github.com/orta).
```
Comment (dblock) [inline CHANGELOG.md]: There should be no period after `suite.`. Also remove `some`, that's too verbose. 

What's `fit` or `fdescribe`?

**Comment by dblock** [comment]
Lets run `rubocop -a Dangerfile`, please. Also add Dangerfile to .rubocop.yml.

File: lib/grape.rb [hunk 2]
```diff
@@ -24,6 +24,8 @@
 
 require 'virtus'
 
+# I will remove this comment in an amended commit once Danger is set up.
```
Comment (dblock) [inline lib/grape.rb]: Please remove this comment ;)

**Comment by orta** [comment]
👍

**Comment by dblock** [comment]
Excellent, merging.

**Comment by dblock** [comment]
Ok, so, how can we make this shareable across the Grape ecosystem?

**Pull request merged**